test/AspNetCoreModule.Test/AspNetCoreModule.Test.csproj

```diff
@@ -20,11 +20,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Testing" Version="$(MicrosoftAspNetCoreTestingPackageVersion)" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="$(MicrosoftNETTestSdkPackageVersion)" />
+    <PackageReference Include="Microsoft.AspNetCore.Testing" Version="2.1.0-preview1-27644" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
     <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="$(SystemRuntimeCompilerServicesUnsafePackageVersion)" />
-    <PackageReference Include="xunit" Version="$(XunitPackageVersion)" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="$(XunitRunnerVisualstudioPackageVersion)" />
+    <PackageReference Include="xunit" Version="2.3.1" />
```

Comment:
Define these versions in dependencies.props.

Reply:
Right now I can't allow korebuild to upgrade these pacakges. The tests don't seem to work with the newer version of xunit and .net test framework. The error is really vague and hard to debug so I am leaving them unchanged. At least for now.

Reply:
File an issue for it then.

Reply:
https://github.com/aspnet/AspNetCoreModule/issues/315

Reply:
https://github.com/aspnet/AspNetCoreModule/issues/316